src/api/entities/Account/MultiSig/types.ts

```diff
@@ -1,8 +1,14 @@
 import BigNumber from 'bignumber.js';
 
+import { MultiSig } from '~/internal';
 import { Signer } from '~/types';
 
 export interface MultiSigDetails {
   signers: Signer[];
   requiredSignatures: BigNumber;
 }
+
+export interface MultiSigSigner {
```

Comment:
```suggestion
export interface MultiSigSigners {
```

To me this feels like it should be plural since its all of the signers for the multi sig